pkg/controller/podautoscaler/horizontal.go

```diff
@@ -39,6 +39,8 @@ const (
 	// TODO: make it a flag or HPA spec element.
 	tolerance = 0.1
 
+	defaultTargetCPUUtilizationPercentage = 80
```

Comment:
Personally I don't like the idea to have it exposed as a configuration option since it's a subject of further changes especially when we will add more options for HPA.

@bgrant0607 @fgrzadkowski @mwielgus any thoughts?